style.css

```diff
@@ -96,3 +106,101 @@ h1 {
   width: 20%;
   float: right !important;
 }
+
+.nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  background-color: rgb(0, 75, 0);
+}
+
+.nav h1 {
+  font-family: 'Fredoka', sans-serif;
+  font-size: xx-large;
+  color: #fff;
+}
+
+.anchors {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+}
+
+.anchors a {
+  font-family: 'Fredoka', sans-serif;
+  font-size: large;
+  padding: 0 15px;
+  color: #fff;
+  cursor: pointer;
+}
+
+.anchors a:hover {
+  color: black;
+}
+
+.book-add h2 {
+  text-align: center;
+  font-family: 'Fredoka', sans-serif;
+  font-family: 'Fredoka', sans-serif;
+  font-size: xx-large;
+}
+
+.contact {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+  gap: 80px;
+}
```

Comment:
- [ ] (Optional) On a large screen, the rest of your contact details are hidden. Kindly add a `padding-bottom` to the `.section` class selector so that it shows the rest of the details. 👍